team2.html

```diff
@@ -33,7 +33,7 @@
                         <li><a href="teamOps.html">TeamOps</a></li>
                         <li><a href="wikiNets.html">WikiNets</a></li> 
                         <li><a href="linuxstar.html">Linuxstar</a></li> 
-                        <li class="active"><a href="team2.html">Team 2</a></li>
+                        <li class="active"><a href="FrontRowSquad.html">Team 2</a></li>

```

Comment:
This only changes the link in the navigation menu of the current page, directing to "FrontRowSquad.html" which does not exist. We could just keep the html file named "team2" and change the labels, or rename the file. Will also need to update the other page's with the new link!